tests/heif/heif_im2im.c

```diff
@@ -0,0 +1,70 @@
+/**
+ * Check if it's any difference between the original bitmap and a encoded and
+ * decoded `4:4:4` HEIF lossless image.
+ */
+
+
+#include "gd.h"
+#include "gdtest.h"
+
+int main()
+{
+	gdImagePtr src, dst;
+	int r, g, b;
+	void *p;
+	int size = 0;
+	int status = 0;
+	CuTestImageResult result = {0, 0};
+
+	src = gdImageCreateTrueColor(100, 100);
+	if (src == NULL) {
```

Comment:
i think these if statements could be collapsed into a single call to `gdTestAssert` or `gdTestAssertMsg`

you'll want to look at the other tests to see if you can collapse them too